README.md

```diff
@@ -29,18 +29,15 @@ have the necessary libraries installed.
 
 ## Governance
 
-This repository is maintained and governed by Pelagicore AB.
-
-Maintainer:
-
-- Richard Pannek <richard.pannek@pelagicore.com>
```

Comment:
I think we should replace the maintainer to someone from the team, not just remove them.

Reply:
What if person. Is on holidays? Is not replying to someones email for six weeks a bad peactice? Also, who the heck submits big reports via email i 2018? 😀

I believe our audience has github profiles 🤔

Reply:
A maintainer is not only for replying to emails, they:
- are the main responsible for that component
- have the best knowledge about where the component should be heading
- have all the background information about the component
- should be looking at each and every pull request to stay updated
- if they are away for a long time and not able to react they should assign a replacement maintainer for that time
- if there is no maintainer, it might happen that nobody feels responsible for a component and it gets neglected and starts to bit rot
- it's best practice that every open source component has a maintainer as the main responsible
- we want to teach the industry about best practicies, to do that we need to do it ourselves

Reply:
Btw. if you look at the list of rpos which are still maintained by us, 8 out of 10 do have a official maintainer mentioned either in the MAINTAINER.md or in the README.